packages/rocketchat-message-snippet/server/requests.js

```diff
@@ -0,0 +1,47 @@
+/* global Cookies */
+WebApp.connectHandlers.use('/snippet/download', function(req, res) {
+	var match = /^\/([^\/]+)\/(.*)/.exec(req.url);
+
+	if (match[1]) {
+		let snippet = RocketChat.models.Messages.findOneById(match[1]);
```

Comment:
Can you verify if the message is a snippet, otherwise users can download any message